predict-daily-temperature/README.md

```diff
@@ -0,0 +1,26 @@
+# Predicting Daily Temperature
+
+<p style="margin:30px">
+    <img style="display:inline; margin-right:50px" width=50% src="https://www.featuretools.com/wp-content/uploads/2017/12/FeatureLabs-Logo-Tangerine-800.png" alt="Featuretools" />
+    <img width=50% src="https://evalml-web-images.s3.amazonaws.com/evalml_horizontal.svg" alt="Featuretools" />
+</p>
+
+This dataset in this problem only contains two colmns--a time index and a target daily temperatures column. Solving this problem as a univariate time series problem lets us build a rich set of features and a useful machine learning model. We can automate the process using both [Featuretools](https://www.featuretools.com) for time series feature engineering and [EvalML](https://github.com/alteryx/evalml) for performing automated time series regression.
+
+We'll demonstrate several end-to-end workflows, starting with a baseline workflow, then moving to one where we build our features using Featuretools, and finally handling everything inside of EvalML.
+
+## Highlights
+
+- Understand what makes a time series problem different from other machine learning problems
+- Show the impact that time series feature engineering has on our model
+- Quickly make end-to-end workflow using time-series data and time series modeling concepts
+- Use AutoMLSearch to perform automated time series machine learning
+
+## Running the tutorial
+
+The data can be found in the `dataset` directory, so the notebook can be run without having to download any additional data.
```

Comment:
Linked to the source data in https://github.com/alteryx/open_source_demos/pull/35/commits/fcf2e0a10e3c81a661b06bbe08232c537600811f, where I also made sure the data was a combination of the original kaggle train and test data since we handle splitting ourselves.